pkgs/servers/home-assistant/component-packages.nix

```diff
@@ -248,7 +248,7 @@
     "ffmpeg" = ps: with ps; [ ha-ffmpeg ];
     "ffmpeg_motion" = ps: with ps; [ ha-ffmpeg ];
     "ffmpeg_noise" = ps: with ps; [ ha-ffmpeg ];
-    "fibaro" = ps: with ps; [ ]; # missing inputs: fiblary3
+    "fibaro" = ps: with ps; [ fiblary3 ];
```

Comment:
Put this in a different commit, because they are about different things even if related.

(It is necessary because of future cherry-pickings and rebases.)